Makefile

```diff
@@ -90,7 +90,7 @@ test-with-cov:
 
 .PHONY: create-github-release
 ci-create-github-release:
```

Comment:
Looks like the phony target and the target don't match up. Can we fix this while we're in here?